src/topology/algebra/module.lean

```diff
@@ -820,7 +820,7 @@ variables [has_continuous_add M₂]
 instance : semimodule S (M →L[R] M₂) :=
 { smul_zero := λ _, ext $ λ _, smul_zero _,
   zero_smul := λ _, ext $ λ _, zero_smul _ _,
-  one_smul  := λ _, ext $ λ _, one_smul _ _,
+  one_smul  := λ _, ext $ λ _, by exact one_smul _ _,
```

Comment:
Are you sure you needed this change? You haven't even touched `one_smul`!

Reply:
I know, but the builds were failing and using `by exact` there fixed them...

Reply:
This is the build error:

```lean
Run leanpkg configure
configuring mathlib 0.1
Error: /home/runner/work/mathlib/mathlib/src/topology/algebra/module.lean:823:31:
synthesized type class instance is not definitionally equal to expression inferred by typing rules, synthesized
  mul_action_with_zero.to_mul_action
inferred
  distrib_mul_action.to_mul_action
Error: Process completed with exit code 1.
```